e2e/pages/events/createOrderEvent.page.js

```diff
@@ -83,9 +83,11 @@ module.exports = {
     I.fillField(this.fields.details, orders[0].details);
   },
 
-  enterJudgeAndLegalAdvisor(judgeLastName, legalAdvisorName, judgeTitle = judgeAndLegalAdvisor.fields.judgeTitleRadioGroup.herHonourJudge) {
+  enterJudgeAndLegalAdvisor(judgeLastName, legalAdvisorName, judgeTitle = judgeAndLegalAdvisor.fields.judgeTitleRadioGroup.herHonourJudge,
+    judgeEmailAddress) {
     judgeAndLegalAdvisor.selectJudgeTitle('', judgeTitle);
     judgeAndLegalAdvisor.enterJudgeLastName(judgeLastName);
+    judgeAndLegalAdvisor.enterJudgeEmailAddress(judgeEmailAddress);
     judgeAndLegalAdvisor.enterLegalAdvisorName(legalAdvisorName);
```

Comment:
What do you think about updating this e2e test to just take 'judge' and 'legalAdvisorName as params? 

Or two separate methods -> enterJudge(Judge) and enterLegalAdvisor(legalAdvisorName)

Reply:
Agreed to create a tech debt ticket for this as requires a bit of work